spec/fixtures/hooks/good_pre_hook

```diff
@@ -2,10 +2,10 @@
 
 echo "input args are $@" > /tmp/hook.out 2>&1
 
-if [[ -n "$OIDC_ACCESS_TOKEN" ]]; then
+if [[ -n "$OOD_OIDC_ACCESS_TOKEN" ]]; then
```

Comment:
I had to rebase to pick up the tests to modify this bit. Just FYI when we go to document that all the env variables will be prefixed with `OOD_`.